### PR TITLE
[Feature] Added Contact Form

### PR DESCRIPTION
**Pull request by KIET7UKE**
Hey, @SaptarshiSarkar12 I have added the contact form to your website. Kindly verifies it.

1. Light Mode
Screenshot (1103)
2. Dark Mode
Screenshot (1104)

### CODE REVIEW

**Comment by KIET7UKE** [comment]
.deploy

**Comment by KIET7UKE** [comment]
Any Issues @SaptarshiSarkar12

**Comment by SaptarshiSarkar12** [comment]
> Any Issues @SaptarshiSarkar12
I don't know why I can't deploy previews of PRs. Can you see the [workflow file](https://github.com/SaptarshiSarkar12/Drifty/blob/master/.github/workflows/preview.yml) and tell me if there is any problem?

**Comment by SaptarshiSarkar12** [comment]
> Any Issues @SaptarshiSarkar12

@KIET7UKE Where does the message comes, when the submit button is pressed?

**Comment by KIET7UKE** [comment]
@SaptarshiSarkar12 Yes, I forgot to mention that you have to just add your mail action tag 
e.g: action = "mailto:yourmailid".

**Comment by SaptarshiSarkar12** [comment]
> @SaptarshiSarkar12 Yes, I forgot to mention that you have to just add your mail action tag e.g: action = "mailto:yourmailid".

Please add that

File: Website/index.html [hunk 1]
```diff
@@ -114,6 +114,21 @@ <h2>Contact Us</h2>
 			<a href="mailto:saptarshi.programmer@gmail.com" type="button"><i class="fa-sharp fa-solid fa-envelope"></i>
 				Email</a>
 		</div>
+		<div class="contact-me" >
+			<section class="contact section" id="contact">
+				<br>
+				<form action="" method="POST">
+					<input type="hidden" name="_subject" value="New Query Arrived">
+					<input name="name" type="text" class="feedback-input" placeholder="Name" required />
+					<input type="hidden" name="_next" value="http://127.0.0.1:5500/thankyou.html">
```
Comment (SaptarshiSarkar12) [inline Website/index.html:123]: I think the below change is also required and `thankyou.html` needs to be created.
```suggestion
					<input type="hidden" name="_next" value="https://saptarshisarkar12.github.io/Drifty/thankyou.html">
```

Comment (KIET7UKE) [reply Website/index.html:123]: Yes check this.

Comment (KIET7UKE) [reply Website/index.html:123]: > > @SaptarshiSarkar12 Yes, I forgot to mention that you have to just add your mail action tag e.g: action = "mailto:yourmailid".
> 
> Please add that

Ok adding

Comment (SaptarshiSarkar12) [reply Website/index.html:123]: saptarshi.programmer@gmail.com

Comment (KIET7UKE) [reply Website/index.html:123]: > 

No that's not mandatory

**Comment by KIET7UKE** [comment]
@SaptarshiSarkar12 I have created slides of steps on how to add this check below:
https://scribehow.com/shared/Get_message_from_visitors__xI1aPzEsR3SWdCcXGz0UWA

File: Website/index.html (hunk 1 above)
Comment (SaptarshiSarkar12) [reply Website/index.html:123]: > > 
> 
> No that's not mandatory

Which one?

**Comment by SaptarshiSarkar12** [comment]
> @SaptarshiSarkar12 I have created slides of steps on how to add this check below: https://scribehow.com/shared/Get_message_from_visitors__xI1aPzEsR3SWdCcXGz0UWA

@KIET7UKE Will this form work if I add "action=mailto:mail_ID"?

File: Website/index.html (hunk 1 above)
Comment (SaptarshiSarkar12) [reply Website/index.html:123]: > > > 
> > 
> > 
> > No that's not mandatory
> 
> Which one?

@KIET7UKE Please tell.

**Comment by KIET7UKE** [comment]
> > @SaptarshiSarkar12 I have created slides of steps on how to add this check below: https://scribehow.com/shared/Get_message_from_visitors__xI1aPzEsR3SWdCcXGz0UWA
> 
> @KIET7UKE Will this form work if I add "action=mailto:mail_ID"?

Yes, It'll work but sometimes won't
The best case is to make an account at https://getform.io/ 
register here with your email address, and simply add the generated form endpoint in the action tag. easy.

**Comment by KIET7UKE** [comment]
@SaptarshiSarkar12 Did this all solved you problem?

**Comment by SaptarshiSarkar12** [comment]
> @SaptarshiSarkar12 Did this all solved your problem?

No @KIET7UKE , getform.io allows upto 50 submissions. i researched and found that we can also use "php" file to send e-mail directly. Here is the article - https://blog.hubspot.com/marketing/html-form-email [method 2]

**Comment by KIET7UKE** [comment]
@SaptarshiSarkar12 Okay, so what are you saying, will I do that PHP connection also and then again make a pr?

Or should I raise another PR for this contact form connection

**Comment by SaptarshiSarkar12** [comment]
> @SaptarshiSarkar12 Okay, so what are you saying, will I do that PHP connection also and then merge into this pr?
> 
> Or should I raise another PR for this contact form connection

Will php work for github pages?

**Comment by SaptarshiSarkar12** [comment]
> > @SaptarshiSarkar12 Okay, so what are you saying, will I do that PHP connection also and then merge into this pr?
> > Or should I raise another PR for this contact form connection
> 
> Will php work for github pages?

@KIET7UKE

**Comment by KIET7UKE** [comment]
> > @SaptarshiSarkar12 Okay, so what are you saying, will I do that PHP connection also and then merge into this pr?
> > 
> > Or should I raise another PR for this contact form connection
> 
> Will php work for github pages?

That's I think will conflict in future

**Comment by SaptarshiSarkar12** [comment]
> > > @SaptarshiSarkar12 Okay, so what are you saying, will I do that PHP connection also and then merge into this pr?
> > > Or should I raise another PR for this contact form connection
> > 
> > 
> > Will php work for github pages?
> 
> That's I think will conflict in future

@KIET7UKE What about this : https://www.youtube.com/watch?v=0X0kovjzLrw ?

**Comment by KIET7UKE** [comment]
> > > > @SaptarshiSarkar12 Okay, so what are you saying, will I do that PHP connection also and then merge into this pr?
> > > > Or should I raise another PR for this contact form connection
> > > 
> > > 
> > > Will php work for github pages?
> > 
> > That's I think will conflict in future
> 
> @KIET7UKE What about this : https://www.youtube.com/watch?v=0X0kovjzLrw ?

This is same as I did earlier with getforms.io

For now keep this things working, later on many contributors will improve this as well.

**Comment by SaptarshiSarkar12** [comment]
> > > > > @SaptarshiSarkar12 Okay, so what are you saying, will I do that PHP connection also and then merge into this pr?
> > > > > Or should I raise another PR for this contact form connection
> > > > 
> > > > 
> > > > Will php work for github pages?
> > > 
> > > 
> > > That's I think will conflict in future
> > 
> > 
> > @KIET7UKE What about this : https://www.youtube.com/watch?v=0X0kovjzLrw ?
> 
> This is same as I did earlier with getforms.io
> 
> For now keep this things working, later on many contributors will improve this as well.

ok @KIET7UKE

**Comment by SaptarshiSarkar12** [comment]
@KIET7UKE Then, what to do with this PR?

**Comment by KIET7UKE** [comment]
> @KIET7UKE Then, what to do with this PR?

Merge it for now, will raise another issue for contact form redirection.
And I will work on this.

**Comment by SaptarshiSarkar12** [comment]
> > @KIET7UKE Then, what to do with this PR?
> 
> @SaptarshiSarkar12 Merge it for now, will raise another issue for contact form redirection. And I will work on this.

ok.

File: Website/index.html [hunk 2]
```diff
@@ -114,6 +114,21 @@ <h2>Contact Us</h2>
 			<a href="mailto:saptarshi.programmer@gmail.com" type="button"><i class="fa-sharp fa-solid fa-envelope"></i>
 				Email</a>
 		</div>
+		<div class="contact-me" >
+			<section class="contact section" id="contact">
+				<br>
+				<form action="" method="POST">
```
Comment (SaptarshiSarkar12) [inline Website/index.html:120]:
```suggestion
				<form action="mailto:saptarshi.programmer@gmail.com" method="POST">
```

File: Website/index.html (hunk 1 above)
Comment (SaptarshiSarkar12) [inline Website/index.html:123]:
```suggestion
```

**Review by SaptarshiSarkar12 — CHANGES_REQUESTED** [review]
@KIET7UKE Please commit the suggested changes

**Comment by KIET7UKE** [comment]
@SaptarshiSarkar12 done

**Comment by SaptarshiSarkar12** [comment]
> @SaptarshiSarkar12 done

@KIET7UKE Don't you think that a form with no action doesn't seem good? I think that I shouldn't introduce the form section as the action hasn't been configured yet.

**Comment by KIET7UKE** [comment]
@SaptarshiSarkar12 Okay so give me some time, I'll make this form working and again commit it

**Comment by SaptarshiSarkar12** [comment]
> > @SaptarshiSarkar12 done
> 
> @KIET7UKE Don't you think that a form with no action doesn't seem good? I think that I shouldn't introduce the form section as the action hasn't been configured yet.

Also, "mailto:" opens the e-mail client , so , I think this issue needs more help from others.

**Comment by SaptarshiSarkar12** [comment]
> @SaptarshiSarkar12 Okay so give me some time, I'll make this form working and again commit it

ok @KIET7UKE .

**Pull request closed without merging**